packages/material-ui/src/Tab/Tab.js

```diff
@@ -122,6 +119,7 @@ const TabWrapper = experimentalStyled('span', {
   justifyContent: 'center',
   width: '100%',
   flexDirection: 'column',
+  lineHeight: 1.25,
```

Comment:
current wrapper inherit `line-height: 1.75` which ~24px make 2 lines looks wierd. using `1.25` make it looks closer to the spec.